tools/rdm/TestDefinitions.py

```diff
@@ -5071,6 +5071,8 @@ def Test(self):
       self.SetNotRun('Unable to determine pin code')
       return
 
+    # Section 3.9, out of range pins return NR_FORMAT_ERROR rather than
+    # NR_DATA_OUT_OF_RANGE like one may expect.
```

Comment:
I'm still not following, can you propose the comment you want to see?

The test is for an out-of-range PIN. Normally out-of-range fields return NR_DATA_OUT_OF_RANGE, in this case we're supposed to return NR_FORMAT_ERROR. That's what the comment is trying to say.